cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -146,6 +207,20 @@ void setStartAndRun(ProgramId id, String pid, long startTime, long runTime,
   Map<ProgramRunId, RunRecordMeta> getRuns(Set<ProgramRunId> programRunIds);
 
   /**
+   * Fetches the active (i.e STARTING or RUNNING or SUSPENDED) run records against a given NamespaceId.
```

Comment:
a RESUMING program will have a record type of `TYPE_RUN_RECORD_STARTED`, so yes it will.